source/_docs/pantheon-yml.md

```diff
@@ -111,6 +72,25 @@ Add `drush_version` to the top level of the `pantheon.yml` file to configure the
 drush_version: 8
 ```
 For more information, see [Managing Drush Versions on Pantheon](/docs/drush-versions).
+
+
+### Custom Upstreams and pantheon.upstream.yml
+Add a `pantheon.upstream.yml` file to your organization's custom upstream to set default settings for all downstream sites. Downstream sites can override the defaults at the site-level with pantheon.yml. Quicksilver workflows are not supported in pantheon.upstream.yml.
+
+Create the `pantheon.upstream.yml` in the root of the custom upstream's codebase. Help avoid merge conflicts by adding pantheon.upstream.yml to the upstream's .gitignore. Example: [Adding pantheon.upstream.yml to Drops-7](https://github.com/pantheon-systems/WordPress/pull/123/files)
+
+Because pantheon.upstream.yml is in the .gitignore use git force-add to commit:
+
+```bash
+$ git add -f pantheon.upstream.yml
+$ git commit -n "Change the default php version for this upstream to 7.0.x"
+```
+
+#### Nested docroot
+Set the `web_docroot` property in  upstream's `pantheon.upstream.yml` file.
```

Comment:
Feels weird that this is the only advanced configuration we call out in context of custom upstreams. Can we add context? I think we need to preserve our original warning describing the implications of overriding this setting downstream.